plugins/kubernetes/app/models/kubernetes/api/pod.rb

```diff
@@ -18,6 +18,10 @@ def namespace
         @pod.dig(:metadata, :namespace)
       end
 
+      def annotations
+        @pod.dig(:metadata, :annotations)
```

Comment:
needs to be `||= {}` or it will blow up when there are no annotations
```suggestion
        @pod[:metadata][:annotations] ||= {}
```

Reply:
Good catch, thanks! Applied and squashed now :)